### ISSUE TITLE
508 Compliance Project

### ISSUE THREAD

**Issue by bowlenj** [ONRR/onrr.gov-site#525]
Please delete this 2019 outdated pdf contact list - https://www.onrr.gov/ReportPay/PDFDocs/operasgn.pdf

Replace Onshore Production Reporting with attached updated pdf
RRM Onshore Production Contacts.pdf
[RRM Onshore Production Contacts.pdf](https://github.com/ONRR/onrr.gov-site/files/6045724/RRM.Onshore.Production.Contacts.pdf)

https://www.onrr.gov/about/contact.htm

**Comment by bowlenj** [comment]
We found an error on the contact list for Onshore Production Contacts.  Please use this updated contact list instead.
[RRM Onshore Production Contacts.pdf](https://github.com/ONRR/onrr.gov-site/files/6050317/RRM.Onshore.Production.Contacts.pdf)

**Comment by bowlenj** [comment]
Please dis-regard this issue for now, more errors were found.  I will submit new issue when I get correct information.  Thank you!

**Comment by mcharg** [comment]
@bowlenj We have an xls version of the onshore reporter contacts that was made accessible.  Should we swap out all the old pdfs with this?
https://www.onrr.gov/ReportPay/PDFDocs/Onshore.Production.Contacts.101620.xlsx

**Comment by bowlenj** [comment]
Please dis-regard this one.  We're wanting our contact lists to be pdf's but the one I provided had some errors in it.  I'm waiting on the supervisors to get the corrections to me.  We can close this issue for now and I will open a new one when I get the corrected document.

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resource Revenue
303-231-3202
Jessica.Bowlen@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.

**Comment by bowlenj** [comment]
That's a good idea, I just realized what you were saying.  Let's swap out the outdated pdf's with the new xcel until we get the updated version to you.

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resource Revenue
303-231-3202
Jessica.Bowlen@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.

**Comment by mcharg** [comment]
@bowlenj I created this issue to replace the links.  If you notice links to pdfs on other pages, please add them to that issue.
https://github.com/ONRR/onrr.gov-site/issues/527

**Comment by bowlenj** [comment]
Will do, thank you!

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resource Revenue
303-231-3202
Jessica.Bowlen@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.
